nncf/openvino/rt_info.py

```diff
@@ -17,6 +17,23 @@
 from nncf.scopes import IgnoredScope
 
 
+def handle_ignored_scope(value: IgnoredScope) -> IgnoredScope:
+    """
+    Dump Ignored Scope parameters more gracefully.
+    https://github.com/openvinotoolkit/nncf/issues/2564
+    Remove keys where value is empty and key is `validate`
+
+    :param value: IgnoredScope instance, based on the quantization parameters
+    """
+    new_value = IgnoredScope(**(value.__dict__))
```

Comment:
Think about how we can use `dataclasses.asdict` method here.
https://docs.python.org/3/library/dataclasses.html#dataclasses.asdict